homeassistant/components/ted5000/sensor.py

```diff
@@ -106,13 +120,13 @@ def update(self) -> None:
 class Ted5000Gateway:
     """The class for handling the data retrieval."""
 
-    def __init__(self, url):
+    def __init__(self, url: str) -> None:
         """Initialize the data object."""
         self.url = url
-        self.data = {}
+        self.data: dict[int, dict[str | None, int | float]] = {}
```

Comment:
```suggestion
        self.data: dict[int, dict[str, int | float]] = {}
```

Reply:
I had to adjust native_value to make this work. Should be ok now.